doc/payment-disclosure.md

```diff
@@ -0,0 +1,107 @@
+# Payment Disclosure (Experimental Feature)
+
+**Summary**
+
+Use RPC calls `z_getpaymentdisclosure` and `z_validatepaymentdisclosure` to reveal details of a shielded payment.
+
+**Who should read this document**
+
+Frequent users of shielded transactions, payment processors, exchanges, block explorer
+
+### Experimental Feature
+
+This is an experimental feature.  Enable it by launching `zcashd` with flags:
+
+    zcashd -experimentalfeatures -paymentdisclosure -debug=paymentdisclosure -txindex=1
+
+These flags can also be set as options in `zcash.conf`.
+
+All nodes that generate or validate payment disclosures must run with `txindex=1` enabled.
```

Comment:
Is this enforced in the code? I don't recall seeing this as an explicit requirement in #2159.

Reply:
Yes, needs it, but not enforced. I filed a ticket to follow-up https://github.com/zcash/zcash/issues/2758